ee/clickhouse/queries/funnels/funnel_correlation.py

```diff
@@ -159,7 +161,7 @@ def get_partial_event_contingency_tables(self) -> List[EventContingencyTable]:
                 countIf(person.steps = target_step) AS success_count,
 
                 -- And the converse being for failures
-                countIf(person.timestamp <> target_step) AS failure_count
+                countIf(person.steps <> target_step) AS failure_count
```

Comment:
This was working fine before, because `funnel_step=2` , so we were only choosing people who completed the funnel 🤷 .

IF we were choosing all people, failure count would've been 2, as expected for the old code.

Reply:
yeah that was so confusing!